src/OmniSharp.Roslyn.CSharp/Services/Signatures/SignatureHelpService.cs

```diff
@@ -90,24 +91,25 @@ public SignatureHelpService(OmniSharpWorkspace workspace)
             // Walk up until we find a node that we're interested in.
             while (node != null)
             {
-                var invocation = node as InvocationExpressionSyntax;
-                if (invocation != null && invocation.ArgumentList.Span.Contains(position))
+   
```

Comment:
My bad, should have been more specific. Blank line between *close brace* and statement :).